src/renderers/webgl-fallback/nodes/GLSLNodeBuilder.js

```diff
@@ -636,9 +637,24 @@ ${ flowData.code }
 
 	}
 
+	getExtension( name, mode, shaderStage = this.shaderStage ) {
```

Comment:
Like mentioned in #28615, it does not make sense when a `get*()` method returns nothing.

This should be `enableExtension()` so it's the pendant to `enableDirective()`.